test/fixtures/users.yml

```diff
@@ -5,3 +5,15 @@ dan:
 
 kari:
```

Comment:
You need to update these two fixtures (Dan & Kari) so that they match the updated user model.